setup.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -e
+set -x
+
+# Create environment:
+conda update -n base conda -y
+conda env create --force -q
+
+# Start environment:
+source activate good
+
+# Update environment (might break stuff. move fast!?)
+conda update --all --yes
```

Comment:
update --all makes me a bit nervous.  I am concerned that this will lead to a poor user experience for the students when the come across buggy code and they might have the skills to quickly debug

Reply:
I understand that point. 

The flip side is not updating our materials and teaching students updated versions of packages. I rather have everyone using the current version and fixing small bugs as we go rather than keep to a pinned version and having massive bugs latter.

For an individual class, there should be a branch with pinned version. 

We can take that line out. I don't feel that strongly.

Reply:
yep, and that is very reasonable.  I think when we are actively developing it should not be a problem since we will catch the bugs ourselves and can quickly correct them.  I am more concerned when we stop developing.

We could leave it as is for now, and then remove it once we finish teaching the course?

Reply:
create a jira ticket to help us remember to remove this when we finish teaching the course 

https://deltanalytics.atlassian.net/browse/LAB-36